stonesoup/types/state.py

```diff
@@ -61,6 +61,9 @@ class StateMutableSequence(Type, MutableSequence):
     def __init__(self, states=None, *args, **kwargs):
         if states is None:
             states = []
+        elif not isinstance(states, list):
```

Comment:
May be better to check if `states` is a `collections.abc.MutableSequence` instead.